src/sidebar/components/toast-messages.js

```diff
@@ -45,6 +44,17 @@ function ToastMessage({ message, onDismiss }) {
         <div className="toast-message__message">
           <strong>{prefix}: </strong>
           {message.message}
+          {message.moreInfoURL && (
+            <div className="toast-message__link">
+              <a
+                href={message.moreInfoURL}
+                onClick={event => event.stopPropagation()}
```

Comment:
This is to keep clicks from dismissing the message; should this have more commenting? Be in a non-anonymous function for clarity?

Reply:
A short inline comment would be useful.